sandbox/src/Home.js

```diff
@@ -1,106 +1,42 @@
 import React from "react";
-import { withRouter } from "react-router-dom";
 import ContentSecurityPolicy from "./components/ContentSecurityPolicy";
-
-let previousPath;
-function HomeWithHistory({ history }) {
-  history.listen(loc => {
-    if (loc.pathname !== previousPath) {
-      const instanceName = loc.pathname.includes("orgTwo")
-        ? "organizationTwo"
-        : "alloy";
-
-      window[instanceName]("sendEvent", {
-        renderDecisions: true,
-        xdm: {
-          eventType: "page-view"
-        }
-      });
+import useSendPageViewEvent from "./useSendPageViewEvent";
+
+const getIdentity = () => {
+  window.alloy("getIdentity", { namespaces: ["ECID"] }).then(function(result) {
+    if (result.identity) {
+      console.log(
+        "Sandbox: Get Identity command has completed.",
+        result.identity.ECID
+      );
+    } else {
+      console.log(
+        "Sandbox: Get Identity command has completed but no identity was provided in the result (possibly due to lack of consent)."
+      );
     }
-    previousPath = loc.pathname;
```

Comment:
Sending the event within a `history.listen` didn't make a lot of sense, because it would only send an event with the user navigated and, I think, only if the user hit Home first (I forget). I changed it so that every view that wants to send an event when the view loads can leverage a `useSendPageViewEvent` module.